WeatherTests/UnitTests/WeatherServiceTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Moq;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using WeatherBL;
+using WeatherBL.Interfaces;
+using WeatherBL.Models;
+using WeatherBL.Validators;
+using WeatherDAL;
+using WeatherDAL.Models;
+
+namespace WeatherTests.UnitTests
+{
+    [TestFixture]
+    public class WeatherServiceTests
+    {       
+        private WeatherService service;
+        private Mock<IWeatherProvider> mockProvider;
+        private Mock<IValidator> mockValidator;
+        private double temp;
+        private string city;
+        
+
+        public WeatherServiceTests()
+        {
+            temp = 300.00;
```

Comment:
if this is common for all methods you can leave it here, if not move to arrange section

Reply:
yeah its common for all methods that's why I left the same without changes